### PR TITLE
fix: pip install doesn't run PostInstallCommand()

### PR DESCRIPTION
**Pull request by Vaydheesh**
- solves #72 
- added targets for develop & egg_info
- bump version to 0.2.1

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,49 +1,62 @@
...
+
+    apertium.installer.install_apertium()
+    apertium.installer.install_module('eng')
+    apertium.installer.install_module('en-es')
+    apertium.installer.install_module('kaz-tat')
+    apertium.installer.install_wrapper('python3-apertium')
+    apertium.installer.install_wrapper('python3-apertium-lex-tools')
+    apertium.installer.install_wrapper('python3-cg3')
+    apertium.installer.install_wrapper('python3-lttoolbox')
+    apertium.installer.install_apertium_linux()
```
Comment (sushain97) [inline setup.py]: Why does this happen regardless of OS? Does the top `install_apertium` not cover this?

Comment (Vaydheesh) [reply setup.py]: `install_apertium` installs only `apertium-all-dev`
`apertium.installer.install_apertium_linux` is for binaries that aren't available on windows

**Comment by Vaydheesh** [comment]
Can we merge this PR

**Comment by sushain97** [comment]
Released on pypi.

**Pull request merged**